superset/assets/javascripts/explore/stores/language.js

```diff
@@ -0,0 +1,24 @@
+
+const $ = window.$ = require('jquery');
+
+export function getLanguage() {
+  function getLocale() {
+    const locale = $.ajax({
+      url: '/superset/rest/api/getLocale',
+      async: false,
+    });
+    return locale.responseText;
+  }
+  switch (getLocale()) {
```

Comment:
Because the English language that the 'API' returns is' en ', but the English translation file's directory is' es'.